FHIR_README.md

```diff
@@ -0,0 +1,78 @@
+![img](./public/images/openemr-rest-api.png)
```

Comment:
If possible my suggestion will be a FHIR logo

Reply:
maybe take openemr logo and add fire image somehow

Reply:
Could also just drop the logo and give the doc a title (same for the API doc)

Reply:
Fixed